Packs/NetBox/Integrations/NetBoxEventCollector/NetBoxEventCollector.py

```diff
@@ -0,0 +1,253 @@
+        return self._http_request(
+            method='GET',
+            url_suffix=url_suffix,
+            full_url=full_url,
+            params=params
+        )
+
+    def search_events(self, url_suffix, limit=DEFAULT_LIMIT, prev_id=0, ordering=''):
+        """
+        Searches for NetBox alerts using the '/<log_type>' API endpoint for log_type in LOG_TYPES.
+        All the parameters are passed directly to the API as HTTP POST parameters in the request
+        Args:
+            limit: int, the limit of the results to return per log_type.
+            prev_id: dict of previous ids that was fetched fot each log_type.
+            ordering: boll, if is True this will return the data starting with the oldest, otherwise, the opposite.
+        Returns:
+            dict: A dict containing the next_run
+            list: A list containing the events
+        """
+        next_id = prev_id
+        results: List[Dict] = []
+
+        next_page = True
+        params = {
+            'limit': limit,
+            'ordering': ordering,
+            'id__gte': next_id,
+        }
+
+        while next_page and len(results) < limit:
+            full_url = next_page if type(next_page) == str else ''
+            response = self.http_request(url_suffix=url_suffix, full_url=full_url, params=params)
+
+            results += response.get('results', [])
+
+            next_page = response.get('next')
+            params = {}
+
+            if results:
+                next_id = results[-1]['id'] + 1
+
+        return next_id, results[:limit]
+
+    def get_first_fetch_id(self, url_suffix, params):
+        """
+        Sets the first fetch ids for log type.
+        Args:
+            url_suffix: str, the log type to fetch.
+            params: dict, the time parameters to fetch.
+        Returns:
+            int: The first fetch id for the log type.
+        """
+        first_log = self.http_request(url_suffix=url_suffix, params={'ordering': 'id', 'limit': 1} | params)
+        try:
+            next_run = first_log.get('results', [{}])[0].get('id')
+        except IndexError:
+            next_run = None
+
+        return next_run
+
+
+def test_module(client: Client) -> str:
+    """
+    Tests API connectivity and authentication'
+    When 'ok' is returned it indicates the integration works like it is supposed to and connection to the service is
+    successful.
+    Raises exceptions if something goes wrong.
+    Args:
+        client (Client): NetBox client to use.
+    Returns:
+        str: 'ok' if test passed, anything else will raise an exception and will fail the test.
+    """
+
+    try:
+        client.search_events(url_suffix=LOG_TYPES[0], limit=1)
+
+    except Exception as e:
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+
+    return 'ok'
+
+
+def get_events(client: Client, limit: int):
```

Comment:
```suggestion
def get_events_command(client: Client, limit: int):
```
Please add return type